static/style.css

```
@@ -0,0 +1,183 @@
+/*Body styling*/
+body {
+  background-image: url("../static/img3.jpg");
+  background-color: rgb(48, 44, 44);
+  height: 1000px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  font-family: Arial;
+  display: flex;
+  flex-flow: row wrap;
+}
+/*Center container styling for login/or signup*/
+.container {
+  background-color: #ffe6d3f1;
+  width: 600px;
+  height: 600px;
```

For this component, since it's not supposed to cover the entire screen, I'd recommend percentage values. Something like this:
```suggestion
  width: 40%;
  height: 75%;
```